src/style.css

```diff
@@ -138,6 +139,54 @@ header {
   color: var(--primary);
 }
 
+.pop {
+  background-color: var(--secondary);
+  margin: 5% auto;
+  height: 80%;
+  width: 100%;
+  top: 0;
+}
+
+.content {
+  padding: 2rem;
+  border: 1px solid #aaa;
+}
+
+.movie-type,
+.movie-details {
+  display: var(--flex);
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.popup-body img {
+  width: max-content;
+  padding: 1rem;
+}
+
```

Comment:
What you have done so far is good 😉 . However, you still need to make your design looks similar to the requirement. 
<img width="507" alt="image" src="https://user-images.githubusercontent.com/35809268/204998174-c69bbb15-e896-43a8-81d9-9f034bde9a4b.png">
Your work :
<img width="785" alt="image" src="https://user-images.githubusercontent.com/35809268/204998391-ac2bd441-391e-4aa8-8d80-c5a2b82a2aee.png">

Reply:
👍 Will get right on it